rest-api-spec/src/main/resources/rest-api-spec/api/search_mvt.json

```diff
@@ -0,0 +1,47 @@
+{
+  "search_mvt": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/search-vector-tile-api.html",
+      "description": "Searches a vector tile for geospatial values. Returns results as a binary Mapbox vector tile."
+    },
+    "stability": "experimental",
+    "visibility": "public",
+    "headers": {
```

Comment:
Can you add the returned content type as well?

Reply:
Mimetype appears to be `application/vnd.mapbox-vector-tile`:

http://www.iana.org/assignments/media-types/application/vnd.mapbox-vector-tile

Reply:
Thanks for the suggestion. Added with 105e36c06398d0ea95d46760e4b87bc4c12fce3f.

@iverase Do you mind double-checking me here?

Per https://github.com/mapbox/vector-tile-spec/issues/48 and http://www.iana.org/assignments/media-types/media-types.xhtml, the MIME type should be `application/vnd.mapbox-vector-tile`.

Reply:
My timing is on point! :P

Reply:
Looks right:
https://github.com/elastic/elasticsearch/blob/bff790011c039f0c8777c55960c6b8141c0aaf3b/x-pack/plugin/vector-tile/src/main/java/org/elasticsearch/xpack/vectortile/rest/RestVectorTileAction.java#L78

Reply:
Yes, that is the mime type we return.